Package.swift

```diff
@@ -118,6 +128,10 @@ let package = Package(
       exclude: [
         "Info.plist",
         "graphql"
+      ],
+      resources: [
+        .copy("graphql/schema.json"),
+        .copy("graphql/schema.graphqls")
```

Comment:
These shouldn't need to be copied into the final resources that go into that framework - they should theoreitcally still get checked out, which is where we'd want to use them.